go-controller/pkg/ovn/libovsdbops/model.go

```diff
@@ -160,6 +168,17 @@ func copyIndexes(model model.Model) model.Model {
 		return &nbdb.NBGlobal{
 			UUID: t.UUID,
 		}
+	case *nbdb.MeterBand:
+		{
```

Comment:
to be consistent, maybe remove the extra { }

Reply:
Ah thanks, fixed!